src/client/Microsoft.Identity.Client/PlatformsCommon/Shared/EventSourcePlatformLogger.cs

```diff
@@ -14,6 +14,11 @@ static EventSourcePlatformLogger()
 
         internal static MsalEventSource MsalEventSource { get; }
 
+        public void LogAlways(string message)
+        {
+            MsalEventSource.Error(message);
```

Comment:
No, we should not log it as an error. Log as is Info. 
The fact that Info(message) calls ` MsalEventSource.Error(message);` is a bug.